pages/establishment/details/views/index.jsx

```diff
@@ -56,14 +57,22 @@ const Index = ({
           </dl>
           <Accordion>
             <ExpandingPanel title={<Snippet>conditions.title</Snippet>}>
-              { establishment.conditions
-                ? (
-                  <Fragment>
-                    <p><Snippet>conditions.hasConditions</Snippet></p>
-                    <ReactMarkdown>{ establishment.conditions }</ReactMarkdown>
-                  </Fragment>
-                )
-                : <p><Snippet>conditions.noConditions</Snippet></p>
+              {
+                <Conditions
+                  conditions={establishment.conditions}
+                  canUpdate={asruUser && !openTask}
```

Comment:
Can we make this dependent on a permissions check or an `allowedActions` or something similar? I'd generally prefer to avoid spreading the logic of _why_ a person can do a thing around the codebase as much as possible.

Reply:
sure, I think it should be in allowedActions